pep-9999.rst

```diff
@@ -259,6 +259,73 @@ No::
         forward_reference: 'OtherClass'
     class OtherClass: ...
 
+Version and Platform Checks
+---------------------------
+
+Type stubs for libraries that support multiple Python versions can use version
+checks to supply version-specific type hints. Type stubs for different Python
+versions should still conform to the most recent Python version's syntax, as
+explain in the Syntax_ section above.
+
+Version checks are if-statements that use ``sys.version_info`` to determine the
+current Python version. Version checks should only check against the ``major`` and
+``minor`` parts of ``sys.version_info``.
+
+Yes::
+
+    if sys.version_info >= (3,):
+      # Python 3-specific type hints.
+    else:
+      # Python 2-specific type hints.
+
+    if sys.version_info.major >= 3:
+      # Same as previous check.
+
+    if sys.version_info[0] >= 3:
+      # This is also the same.
+
+    if sys.version_info >= (3, 5):
+      # Specific minor version features can be easily checked with tuples.
+
+    if sys.version_info < (3,):
+      # This is only necessary when a feature has no Python 3 equivalent.
+      # "<= (2, 7)" does not work because e.g. (2, 7, 1) > (2, 7).
+
+No::
+
+    if sys.version_info.major != 2:
+      # This is less clear than ">= 3".
+
+    if sys.version_info >= (3, 5, 1):
```

Comment:
We should provide guidance on what  to do instead (always write stubs for the latest patch version).